src/GraphWebhooks/Graph/ChangeNotificationEncryptedContentExtensions.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Microsoft.Graph.Models;
+using Microsoft.Kiota.Abstractions.Serialization;
+
+namespace GraphWebhooks;
+
+// This extension is to workaround this bug in the SDK:
+// https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/2237
+// Should no longer be needed after that bug is fixed.
+// This is a direct copy of
+// https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/dev/src/Microsoft.Graph.Core/Extensions/IDecryptableContentExtensions.cs
+public static class ChangeNotificationEncryptedContentExtensions
+{
+    public static async Task<T?> DecryptAsync<T>(
+        this ChangeNotificationEncryptedContent encryptedContent,
+        Func<string, string, Task<X509Certificate2>> certificateProvider) where T : IParsable, new()
+    {
+        if (certificateProvider == null)
+            throw new ArgumentNullException(nameof(certificateProvider));
+
+        var stringContent = await encryptedContent.DecryptAsync(certificateProvider).ConfigureAwait(false);
+        using var contentStream = new MemoryStream(Encoding.UTF8.GetBytes(stringContent));
+        var parseNodeFactory = ParseNodeFactoryRegistry.DefaultInstance;
```

Comment:
we have helper methods from that now. KiotaJsonSerializer.Deserialize()...

Reply:
Nice!